tests/unit/components/NcAppNavigation/NcAppNavigationCaption.spec.ts

```diff
@@ -0,0 +1,74 @@
+import { describe, expect, test } from 'vitest'
+import { shallowMount } from '@vue/test-utils'
+import NcAppNavigationCaption from '../../../../src/components/NcAppNavigationCaption/NcAppNavigationCaption.vue'
+
+describe('NcAppNavigationCaption.vue', () => {
+	test('attributes are passed to actions', async () => {
+		const wrapper = shallowMount(NcAppNavigationCaption, {
+			propsData: {
```

Comment:
If I'm not missing something:

```suggestion
			props: {
```

Reply:
While `propsData` works, yes `props` would be better :)

Reply:
Fixed